internal/pkg/mapper/orchestrator/local_load.go

```diff
@@ -18,8 +18,11 @@ func (m *orchestratorMapper) OnObjectsLoad(event model.OnObjectsLoadEvent) error
 	errors := utils.NewMultiError()
 	for _, object := range event.NewObjects {
 		// Object must be orchestrator config
-		if ok, err := m.isOrchestratorConfig(object); err != nil || !ok {
-			return err
+		if ok, err := m.isOrchestratorConfig(object); err != nil{
+			errors.Append(err)
+			continue
```

Comment:
A tu ma byt `continue` a nie `return`, ... v pripade chyby/alebo ak to nie je orchestrator chceme pokracovat iteraciu nad dalsimi objektami.